src/holochainClient.js

```diff
@@ -113,15 +113,20 @@ let holochainClient
 
 async function initAndGetHolochainClient () {
   if (holochainClient) return holochainClient
+  let url
   try {
-    let url = process.env.NODE_ENV === 'production' ? ('wss://' + window.location.hostname + '/api/v1/ws/') : process.env.REACT_APP_DNA_INTERFACE_URL
-    // Construct url with query param X-Hpos-Admin-Signature = signature
-    const urlObj = new URL(url)
-    const params = new URLSearchParams(urlObj.search.slice(1))
-    params.append('X-Hpos-Admin-Signature', await signPayload('get', urlObj.pathname))
-    params.sort()
-    urlObj.search = params.toString()
-    url = urlObj.toString()
+    if (process.env.REACT_APP_HOLOFUEL_APP === 'true') {
```

Comment:
Can we rename this `REACT_APP_HOLOFUEL_APP` env var?
Because its use is unclear.

Reply:
:1: Good point, this name is a bit outdated now that HoloFuel is bundled within hp-admin when served through the HPOS, and is really only a pure "Holofuel" app whenever run within a Holochain environment. 

Considering this distinction, I purpose renaming it to : `REACT_APP_HOLOCHAIN_APP`.  Thoughts?

Reply:
Its is more like `holofuel in holochain mode`.

Reply:
Ok decided on `REACT_APP_HOLOFUEL_AS_HOLOCHAIN_APP` and pushed that up in the last update.